crates/oxc_language_server/src/main.rs

```diff
@@ -240,34 +240,92 @@ impl LanguageServer for Backend {
                 .iter()
                 .find(|r| r.diagnostic.range == params.range && r.fixed_content.is_some())
             {
-                let title =
-                    report.diagnostic.message.split(':').next().map_or_else(
-                        || "Fix this problem".into(),
-                        |s| format!("Fix this {s} problem"),
-                    );
-
+                let rule_name: Option<String> = None;
```

Comment:
Needs to get the exact rule name.  Is used as part of the disable comment like `eslint-disable-next-line no-debugger`.